modules/core/src/smithy4s/http/HttpUri.scala

```diff
@@ -24,7 +24,7 @@ final case class HttpUri(
       * A sequence of URL-decoded URI segment.
       */
     path: IndexedSeq[String],
-    queryParams: Map[String, Seq[String]],
+    queryParams: Map[String, Seq[Option[String]]],
```

Comment:
I think we should take advantage of the fact we're changing the model to change it to `queryParams: IndexedSeq[(String, Option[String])]`

Reply:
I changed it to `queryParams:IndexedSeq[(String, Option[String])]`, but only in `HttpUri` and left other places like `Metadata` as a map.